winpr/include/winpr/stream.h

```diff
@@ -180,6 +180,12 @@ static INLINE void Stream_Write_UINT8(wStream* _s, UINT8 _v)
 	*_s->pointer++ = (UINT8)(_v);
 }
 
+static INLINE void Stream_Write_INT16(wStream* _s, INT16 _v)
```

Comment:
Not necessary, the `Stream_Write_UINT16`   does the same ;)

Reply:
Yes but that takes a UINT16 and I don't wanted to introduce new warnings/casts

Reply:
true.